sirepo/package_data/auth_role_moderation/approve_email.jinja

```diff
@@ -1,3 +1,4 @@
-Your request for {{ app_name }} access has been approved.
+Your request for Sirepo {{ role_friendly_name }} access has been approved.
```

Comment:
Now I'm confused, because there is "app_name" above which isn't app name any more(?).

Reply:
b1f5dad21